junit-jupiter-api/src/main/java/org/junit/jupiter/api/Assertions.java

```diff
@@ -870,6 +871,51 @@ public static void assertIterableEquals(Iterable<?> expected, Iterable<?> actual
 		AssertIterableEquals.assertIterableEquals(expected, actual, messageSupplier);
 	}
 
+	// --- assertLinesMatch ----------------------------------------------------
+
+	/**
+	 * <em>Asserts</em> that {@code expected} list of {@linkplain String}s matches {@code actual}
+	 * list.
+	 *
+	 * <p>This method differs from other assertions that effectively only check {@link String#equals(Object)},
+	 * in that it uses the following staged match-making algorithm:
```

Comment:
Check. Borrowed the word from [here](https://us.battle.net/forums/en/overwatch/topic/20745144780#post-11).